src/View/Widgets/ProgressInfoWidget.vala

```diff
@@ -83,13 +85,14 @@ public class Marlin.Progress.InfoWidget : Gtk.Grid {
             cancelled (info);
             button.sensitive = false;
         });
-
-        info.bind_property ("status", status, "label");
     }
 
     private void update_data () {
+        status.set_markup (
+            Markup.printf_escaped ("<span font_features='tnum'>%s</span>", info.status)
```

Comment:
Shouldn't this be `<span font-features='tnum' ....`?  It doesn't seem to work otherwise.